src/uikit.cs

```diff
@@ -277,21 +281,27 @@ interface NSDataAsset : NSCopying
 		NSString TypeIdentifier { get; }
 	}
 
+	delegate void NSFileProviderExtensionFetchThumbnailsHandler (NSString identifier, [NullAllowed] NSData imageData, [NullAllowed] NSError error);
+
 	[NoWatch]
 	[NoTV]
 	[iOS (8,0)]
 	[ThreadSafe]
 	[BaseType (typeof (NSObject))]
 	partial interface NSFileProviderExtension {
+		[Deprecated (PlatformName.iOS, 11, 0, message: "Use 'NSFileProviderManager' instead.")]
```

Comment:
This is intended, unfortunately all this interface is full of ugly dot indentation